danish/haervejen/cykeltur/cykeltur.md

```diff
@@ -6,3 +6,7 @@ Dagen er nu ved at være på hæld, og du kan ikke se noget der kunne ligne et
 gæstgiveri i nærheden.
 
 Hvad gør du?
+
+[Holder en fed provinsfest for dig selv](https://www.youtube.com/watch?v=9_xIoJzZtKg).
+
+[Går i panik, smider cyklen og bryder ud i en erotisk, dog klodsede dans sammen med en tilfældig forbipasserende](https://www.youtube.com/watch?v=nYbI07YBFuI).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.